integration_test/dapp_tests/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "uniswapv3-deployment",
```

Comment:
nit: can we change this name and description to make it more generic since it's for all the tests?